shark_turbine/kernel/ops/wave_ops.py

```diff
@@ -143,6 +167,40 @@ def _add_proxy_to_graph(self, region_graph: RegionGraph):
             kwargs={},
         )
 
+    def update_arg(self, idx_or_name: int | str, value: Any):
+        """
+        Update the value of an argument in the node while keeping the
+        underlying fx.Node consistent.
+        """
+        dataclass_fields = fields(self)[5:]
```

Comment:
Can you add a comment about the magic number 5 here? Also, looks like we only use the name member variable of  dataclass_fields so we could just do
```
field_names = [x.name for x in fields(self)[5:]]
```

Reply:
Thanks for pointing me to this again. This magic number skips over the inherited fields from the `CustomOp` superclass. The problem is that this number needs to be adjusted when new fields are added to the superclass. This bugged me a few times already so I implemented this now without the need for a magic number.

Thanks for the `field_names` suggestion 👍